src/main/java/org/example/czy_dojade_backend/models/User.java

```diff
@@ -0,0 +1,50 @@
+package org.example.czy_dojade_backend.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "app_user")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @Column(name = "username")
+    private String username;
+
+    @Column(name = "password")
```

Comment:
Dopiero teraz o tym pomyślałem ale powinniśmy zrobić hashowanie haseł, więc fajnie zmienić to na hash_pswd np.